src/tokens/Assets.test.tsx

```diff
@@ -336,6 +342,37 @@ describe('AssetsScreen', () => {
     expect(navigate).toHaveBeenCalledWith(Screens.DappShortcutsRewards)
   })
 
+  it('does not render Import Token when feature flag is off', () => {
+    mockEnabledFeatureFlags()
+    const store = createMockStore(storeWithPositions)
+
+    const component = (
+      <Provider store={store}>
+        <MockedNavigator component={AssetsScreen} />
+      </Provider>
+    )
+    const { queryByText } = render(component)
+    const button = queryByText('assets.importToken')
+
+    expect(button).toBeNull()
+  })
+
+  it('clicking Import Token opens a screen', () => {
+    mockEnabledFeatureFlags(StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW)
```

Comment:
```suggestion
    jest
      .mocked(getFeatureGate)
      .mockImplementation(
        (gate: StatsigFeatureGates) => gate === StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW
      )
```
code golf. your implementation works, but there are other tests that use this other pattern. i think it is quite readable (it is nice to only see the override variables needed for the test, rather than having to also declare the default state as in the test above). unless there is a clear benefit, i think predictability and consistency are the most important principles so let's reuse the pattern? remember to add `jest.mocked(getFeatureGate).mockReset()` to the `beforeEach` so that the mocks are cleared from previous tests with this approach